daemon/status.go

```diff
@@ -125,7 +129,16 @@ func (h *getHealthz) Handle(params GetHealthzParams) middleware.Responder {
 	d := h.daemon
 	d.statusCollectMutex.RLock()
 	sr := d.statusResponse
+	timestamp := d.statusResponseTimestamp
 	d.statusCollectMutex.RUnlock()
+
+	if time.Since(timestamp) > staleTimeout {
```

Comment:
@aanm you can't just approve like that!! Was this line clear? In the other PR I had a more complicated time check but you asked for it to be commented. Should this have a comment too?

Reply:
I think the code is pretty self explanatory. :100: